### ISSUE TITLE
PS4 account activation with specific ID?

### ISSUE THREAD

**Issue by god-jester** [bucanero/apollo-ps4#7]
Am I missing something? I just tried out Apollo after months of OfflineAccountActivator + Save Mounter working (more or less) just fine. The big attraction is to choose the ID you assign to an offline account. I looked at the docs and all available button controls, and don't see an option for anything besides a randomly generated ID. Is there a point to that?

I hope I'm not coming off as snarky, I might just not know where to look.

edit for use case: 
I'm making a save from scratch for someone. They don't send me a savegame because I don't need it. They send me just their PSN ID. I make a new offline user account, use the Offline Account Activator to set their specific ID on that user, reboot and I'm set. I can copy the save to USB and they can copy it to their legit console. 

It seems I can't do this with Apollo

**Comment by bucanero** [comment]
Right now the account activation is done with a default fake account ID, there's no option to set a user-defined account ID. This enhancement could be added in future versions.

**Comment by bucanero** [comment]
This feature will be available in v1.0.0. For custom account-ids, it will require to have a `owners.xml` file with the account details. File should be stored in `/data/apollo/owners.xml`
Example:

```xml
<?xml version="1.0" encoding="utf-8"?>
<apollo platform="PS4" version="1.0.0">
<owner name="Some User">
<console idps="" psid="FFFFFFFFFFFFFFFF FFFFFFFFFFFFFFFF" />
<user id="10000008" account_id="0123456789abcdef" />
</owner>
<owner name="Another User">
<console idps="" psid="FFFFFFFFFFFFFFFF FFFFFFFFFFFFFFFF" />
<user id="a0000008" account_id="abcdef0123456789" />
</owner>
<owner name="Third User Account">
<console idps="" psid="FFFFFFFFFFFFFFFF FFFFFFFFFFFFFFFF" />
<user id="1000000a" account_id="0123456789aaddff" />
</owner>
</apollo>
```
